style.css

```diff
@@ -261,3 +272,60 @@ html {
 hr {
   background: #dfe1e6;
 }
+
+.contact {
+  background-color: #6070ff;
+  display: flex;
+  border-top-left-radius: 73px;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  background-image: url(images/contactbg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.contact h2 {
+  font-weight: 700;
+  font-size: 38px;
+  line-height: 52px;
+  color: #fff;
+  margin-top: 74px;
+  padding: 11px;
+}
+
+.contact p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  margin: 16px 30px;
+}
+
+.contact form {
+  margin: 24px;
+}
```

Comment:
- [ ] It's better to use `flexbox` to align the elements of the form as it will help with responsivity, kindly do that. 😁